webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -506,6 +507,11 @@ export default class SuggestionBox extends React.PureComponent {
                 }
             }
         }
+
+        if (e) {
+            e.stopPropagation();
```

Comment:
necessary for preventing the enter event key to be capture by the save the changes panel when selecting an emoji with the enter keypress.